### PR TITLE
docs: change link to the styles configuration guide

### PR DESCRIPTION
**Pull request by stefanoww**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [X] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [X] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: #41362

## Does this PR introduce a breaking change?

- [ ] Yes
- [X] No

### CODE REVIEW

**Comment by va-stefanek** [comment]
@googlebot I fixed it..

**Comment by va-stefanek** [comment]
@googlebot I consent.

File: aio/content/guide/component-styles.md [hunk 1]
```diff
@@ -244,7 +244,7 @@ When building with the CLI, you must configure the `angular.json` to include _al
 
 Register **global** style files in the `styles` section which, by default, is pre-configured with the global `styles.css` file.
 
-See the [CLI wiki](https://github.com/angular/angular-cli/wiki/stories-global-styles) to learn more.
+See the [Styles configuration guide]( https://angular.io/guide/workspace-config#styles-and-scripts-configuration) to learn more.
```
Comment (jessicajaniuk) [inline aio/content/guide/component-styles.md:247]: There's a space before the url here. Also can you change it to be a relative link instead of absolute?

**Review by jessicajaniuk — CHANGES_REQUESTED** [review]

**Review by jessicajaniuk — APPROVED** [review]
LGTM 🍪 

Thank you!

**Review by gkalpak — APPROVED** [review]
Thx for fixing this, @stefanoww :tada:
Can you please update the commit message to be `fix(docs): ...` (instead of `fix(docs-infra): ...`)? (`docs-infra` implies a fix to the angular.io app or the docs generation tooling.)

**Comment by stefanoww** [comment]
@gkalpak done :)

**Comment by bartvanraaij** [comment]
This is incomplete. 
There are two more links/reference to the "CLI wiki" is on the page.

**Review by bartvanraaij — CHANGES_REQUESTED** [review]
There are two more links/references to the "CLI wiki" on the page, which should also be changed:
https://github.com/angular/angular/pull/41516/files#diff-2cb9e3390a5e180deefdb0ee86441ed577c65f376d52cc060c4901ddba88fd1aR223

https://github.com/angular/angular/pull/41516/files#diff-2cb9e3390a5e180deefdb0ee86441ed577c65f376d52cc060c4901ddba88fd1aR269

Edit: I'm sorry, I'm not sure how suggesting changes works, this doesn't seem to be the way.
The links are at lines 223 and 269 of the file.

**Comment by stefanoww** [comment]
@bartvanraaij 

You are right, there are some other, thanks for catch that :+1: 

One is:
https://github.com/angular/angular-cli/wiki/angular-workspace
Maybe should be replaced with that one - https://angular.io/guide/workspace-config#angular-workspace-configuration

Second: 
https://github.com/angular/angular-cli/releases - But its works fine, should that be changed?

**Comment by gkalpak** [comment]
@googlebot I consent.

**Comment by gkalpak** [comment]
Sorry, @stefanoww, I somehow missed your last comment (https://github.com/angular/angular/pull/41516#issuecomment-820357473) until just now 😇

I realized my previous commit message suggestion was wrong (it should be `docs: ...` not `fix(docs): ...`), so I checked out the PR to fix it and I also pushed a commit to fix the remaining links in "Component styles".

If there are more references to the CLI wiki, we should definitely fix them (in separate PRs). Feel free to create issues (or better yet PRs) for fixing any other references you find 🙏

**Pull request closed without merging**